### DISCUSSION TITLE
Touch with palm to turn off screen

### DISCUSSION THREAD

**Discussion by Robin-Sch** [InfiniTimeOrg/InfiniTime#2017] [Ideas]
I think it'd be nice if you could turn off the screen by pressing the screen with the palm of your hand (like an apple watch) instead of having to press the power button.

**Comment by mark9064** [comment]
As far as I know this gesture is not supported by the touch panel so unfortunately it is not possible to implement. It would be very difficult to tell between someone tapping the screen strongly and a palm even if accelerometer data was used as in both cases a single touch point would be registered

**Reply by Robin-Sch** [reply]
Ah okay, thanks for your explanation

**Comment by tituscmd** [comment]
Would it be possible to achieve this by tracking if (almost) every part of the screen has been triggered?

**Reply by Robin-Sch** [reply]
Correct me if I'm wrong but like Mark said in the last comment this is not possible due to the way the display works. If you press/tap the screen, no matter with a finger or your whole hand, then only a single point would be registered as "pressed". So we can not check this by checking if every part of the screen has been triggered, as it'd only give a single point rather than an area/multiple points.

**Reply by tituscmd** [reply]
Okay, I didn't know if the comment implied exactly that. Thanks for the elaboration! 😄 

Edit: I somehow skipped the last sentence "a single touch point would be registered"
